testing/run_tests.py

```diff
@@ -919,8 +922,7 @@ def gather_dart_package_tests(build_dir, package_path, extra_opts):
     #
     # Until then, assert that no extra_opts are passed and explain the limitation.
     assert not extra_opts, '%s uses package:test and cannot use CLI args' % package_path
-    # TODO(https://github.com/flutter/flutter/issues/154263): Restore `--disable-dart-dev`.
-    opts = ['test']
```

Comment:
Did you intend to add `--disable-dart-dev` since you removed the TODO and the bug has been fixed?

Reply:
No apologies, I should have clarified - it turns out that flag is no longer needed:
https://github.com/flutter/flutter/issues/154268

Reply:
Gotcha, thanks for the link